modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -271,77 +271,9 @@ command line with the `--ignition-url=` option. In both cases, only HTTP and HTT
 protocols are supported.
 +
 
-* **Live install Ignition config**: This type must be created manually and should be avoided if possible, as it is not supported by Red Hat. With this method, the Ignition config passes to the live install medium, runs immediately upon booting, and performs setup tasks before and/or after the {op-system} system installs to disk. This method should only be used for performing tasks that must be done once and not applied again later, such as with advanced partitioning that cannot be done using a machine config.
+* **Live install Ignition config**: This type can be created using the `coreos-installer` `customize` subcommand and its various options. With this method, the Ignition config passes to the live install medium, runs immediately upon booting, and performs setup tasks before or after the {op-system} system installs to disk. This method should only be used for performing tasks that must be done once and not applied again later, such as with advanced partitioning that cannot be done using a machine config.
```

Comment:
I see the difficulty in `coreos-installer` `customize` here, as I know the `customize` subcommand comes in 2 flavors (iso/pxe). So just a note to say that I considered how we might depict more precisely but the way you have it is the most succinct and least confusing. :)

```suggestion
* **Live install Ignition config**: This type can be created by using the `coreos-installer` `customize` subcommand and its various options. With this method, the Ignition config passes to the live install medium, runs immediately upon booting, and performs setup tasks before or after the {op-system} system installs to disk. This method should only be used for performing tasks that must be done once and not applied again later, such as with advanced partitioning that cannot be done using a machine config.
```